src/stores/mission.ts

```diff
@@ -30,6 +31,28 @@ export const useMissionStore = defineStore('mission', () => {
 
   const currentPlanningWaypoints = reactive<Waypoint[]>([])
 
+  const surveys = reactive<Survey[]>([])
+
+  const paths = reactive<Path[]>([])
```

Comment:
Isn't a path just an array of lat-longs? If so, can we use a GeoJSON type and not import Leaflet here?